database/getters.go

```diff
@@ -33,3 +41,33 @@ func (g *Get) AllTableNames() ([]string, error) {
 
 	return tableNames, nil
 }
+
+func (g *Get) TableColumns(t string) ([]ColumnInfo, error) {
+	queryString := fmt.Sprintf("PRAGMA table_info(%s);", t)
```

Comment:
I belive we have to use [Prepare](https://pkg.go.dev/database/sql#DB.Prepare) for passing variables. This looks good for now, but we should look into this sooner or later

Reply:
added it to the code